### ISSUE TITLE
Layouts, Columns, Sorting order - please reconsider the whole (but easier!)

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#292]
I think it's good for their health that developers don't have a too long period without a serious Issue :O)

**Right now we're stuck with buggy Layouts/column changes:**

* either in settings you disabled them to be saved (like me), then:
a) it's almost impossible to simply remove a column (because it doesn't save the file, as it's considered not changed) - so you've to ALSO change something,
b) or you apply a Layout to one file, and it asks to be saved because it ignores this setting (that's a bug)

* or, you have settings enabling saved Layouts, and everytime you change anything, it changes the file, and the file date - which I consider is bad for accounting, memory, or to locate a problem in your shop stock based on file dates...

**The “good” question, is - IMO -, why would the Layout be file specific?
Then arrives the easy and natural solution!**

I mean:
* if I want the price column to be located BEFORE the quantity, why would it be very specific to *this* file, and not the others?
* or, if I sort by Part ID, why in this other file I would like to *keep it forever* sorted by Color instead?

In short:

**I think Layout, Sort - just like Filters, window dimension and position, like all those defaults should be APP specific and not FILES specific.**

Then everything becomes natural:

* remove the settings/option “save my layout/sorting”: it's always saved for the application

* always save those Layouts, sort and etc for the application

* never modify the file, don't count a layout change as an undoable action: it does NOT concern the file.

And then the problems above all disappear.
:-)
